config/Rules.am

```diff
@@ -34,6 +39,13 @@ AM_CFLAGS += -include $(top_srcdir)/include/os/freebsd/spl/sys/ccompile.h
 AM_CFLAGS += -I/usr/include -I/usr/local/include
 endif
 
+if BUILD_MACOS
+#AM_CFLAGS += ${NO_UNUSED_BUT_SET_VARIABLE} ${DEBUG_CFLAGS} -Wno-tautological-constant-out-of-range-compare
+#AM_CFLAGS += -D_GNU_SOURCE -D__EXTENSIONS__ -D_REENTRANT
+#AM_CFLAGS += -D_POSIX_PTHREAD_SEMANTICS -D_FILE_OFFSET_BITS=64
+#AM_CFLAGS += -D_LARGEFILE64_SOURCE
```

Comment:
Ah yep - I wanted to copy everything as-is, commented out, then only bring them in if they "really were needed". I will trim. As it turns out, we have to have separate Makefile for kernel anyway - since Linux defines "Makefile.in" and we have Makefile.am, so we hooked into their Makefile.in to execute ours (like BSD does with Makefile.bsd). So whole section can go away.